apps/shared/avifpng.c

```diff
@@ -457,8 +454,8 @@ avifBool avifPNGRead(const char * inputFilename,
     }
 
     const int numChannels = png_get_channels(png, info);
-    if ((numChannels != 3) && (numChannels != 4)) {
-        fprintf(stderr, "png_get_channels() should return 3 or 4 but returns %d.\n", numChannels);
+    if (numChannels < 1 || numChannels > 4) {
+        fprintf(stderr, "png_get_channels() should return 1,2,3 or 4 but returns %d.\n", numChannels);
```

Comment:
There could be spaces after the commas, but I'm not sure if there's any consistency in the code base

Reply:
```
git grep "\"[^\"]*, [^\"]*\"" ./src/ | wc -l
63
git grep "\"[^\"]*,[^ ][^\"]*\"" ./src/ | wc -l
2
```
Let's go with a space